resources/i18n/qqq.json

```diff
@@ -45,5 +45,6 @@
 	"monitor-dir-dialog-msg": "\"Monitor\" means \"watch\" in this context. The monitor directory is monitored/watched for new ZIM files.",
 	"monitor-clear-dir-dialog-title": "\"Monitor\" means \"watch\" in this context. The monitor directory is monitored/watched for new ZIM files.",
 	"monitor-clear-dir-dialog-msg": "\"Monitor\" means \"watch\" in this context. The monitor directory is monitored/watched for new ZIM files.",
-	"open-book": "\"Open\" is a imperative, not an adjective."
+	"open-book": "\"Open\" is a imperative, not an adjective.",
+	"open-previous-tabs-on-startup": "The tabs that were open when the user closed the application is opened again when the application is restarted."
```

Comment:
The message key in `qqq.json` doesn't match the key in `en.json` (and the code).

Reply:
Oops, done.